### DISCUSSION TITLE
Allow capital letters in autodiscoverFilter

### DISCUSSION THREAD

**Discussion by MariusKl** [renovatebot/renovate#23158] [Suggest an Idea]
I have a repository "repo" in a project "MY-PROJECT" (bitbucket server). I am able to scan it via
`repositories: ["MY-PROJECT/repo"],`

Afterwards I switched to autodiscovery
`autodiscoverFilter: ["/MY-PROJECT/.*/"],`
but "repo" has not been fetched. I figured out that I am not able to use capitalize letters in autodiscoverFilter.

`autodiscoverFilter: ["/my-project/.*/"],`
worked in the end.

Could you please add support for capitalize letters in autodiscoverFilter? If this is not possible, please add at least some hint in the documentaton.

Thank you.

**Comment by rarkins** [comment]
Capital letters should not be the problem.

**Reply by rarkins** [reply]
But if Bitbucket returns lower case from the API then it means you have no choice but to match against lower case
